lib/foreman_maintain/concerns/os_facts.rb

```diff
@@ -0,0 +1,54 @@
+module ForemanMaintain
+  module Concerns
+    module OsFacts
+      OS_RELEASE_FILE = '/etc/os-release'.freeze
```

Comment:
According to https://www.freedesktop.org/software/systemd/man/os-release.html, there is a fallback location at `/usr/lib/os-release` - should we add support for that too?

Reply:
For sure, pushed the change for same!